src/cmd-generate-release-meta

```diff
@@ -145,6 +145,14 @@ def append_build(out, input_):
         # remove the url as we haven't decided to expose that information publicly yet
         arch_dict["media"]["gcp"]["image"].pop("url")
 
+    # Azure: https://github.com/coreos/stream-metadata-go/issues/13
+    inputaz = input_.get("azure")
+    if inputaz is not None:
+        rhcosext = arch_dict.setdefault("rhel-coreos-extensions", {})
+        rhcosext["azure-image"] = {
```

Comment:
This doesn't follow the nesting structure of the `media` section; I assume this is intentional, in that `azure-image` is a `rhel-coreos-extension`?

`image` could imply a VM image.  Would something like `disk` be clearer?

Reply:
Right, it's an extension per https://github.com/coreos/stream-metadata-go/pull/16
This is now called "disk" too.